From 2e1d593f8ddb6f2b315a8e9696715b80845fdbfb Mon Sep 17 00:00:00 2001
From: Jonathan Calmels <jbjcalmels@gmail.com>
Date: Thu, 29 Jan 2015 15:11:25 +0100
Subject: [PATCH 1/5] input: add torrent and magnet schemes

---
 src/input/item.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/input/item.c b/src/input/item.c
index d357c25..a4f634d 100644
--- a/src/input/item.c
+++ b/src/input/item.c
@@ -999,7 +999,7 @@ input_item_t *input_item_Copy( input_item_t *p_input )
 
 struct item_type_entry
 {
-    const char psz_scheme[7];
+    const char psz_scheme[8];
     uint8_t    i_type;
     bool       b_net;
 };
@@ -1044,6 +1044,7 @@ static int GuessType( const input_item_t *p_item, bool *p_net )
         { "jack",   ITEM_TYPE_CARD, false },
         { "linsys", ITEM_TYPE_CARD, false },
         { "live",   ITEM_TYPE_STREAM, true }, /* livedotcom */
+        { "magnet", ITEM_TYPE_STREAM, true },
         { "mms",    ITEM_TYPE_STREAM, true },
         { "mtp",    ITEM_TYPE_DISC, false },
         { "ofdm",   ITEM_TYPE_CARD, false },
@@ -1063,6 +1064,7 @@ static int GuessType( const input_item_t *p_item, bool *p_net )
         { "svcd",   ITEM_TYPE_DISC, false },
         { "tcp",    ITEM_TYPE_STREAM, true },
         { "terres", ITEM_TYPE_CARD, false }, /* terrestrial */
+        { "torrent",ITEM_TYPE_STREAM, true },
         { "udp",    ITEM_TYPE_STREAM, true },  /* udplite too */
         { "unsv",   ITEM_TYPE_STREAM, true },
         { "usdigi", ITEM_TYPE_CARD, false }, /* usdigital */
-- 
2.1.4

