From 14d44bb9fb3bf128c0483ef3982bd027eca10894 Mon Sep 17 00:00:00 2001
From: Jonathan Calmels <jbjcalmels@gmail.com>
Date: Thu, 29 Jan 2015 15:12:28 +0100
Subject: [PATCH 2/6] input: fix indentation

---
 src/input/item.c | 104 +++++++++++++++++++++++++++----------------------------
 1 file changed, 52 insertions(+), 52 deletions(-)

diff --git a/src/input/item.c b/src/input/item.c
index 65bee8b..0db6b93 100644
--- a/src/input/item.c
+++ b/src/input/item.c
@@ -939,58 +939,58 @@ static int GuessType( const input_item_t *p_item )
     static const struct item_type_entry tab[] =
     {   /* /!\ Alphabetical order /!\ */
         /* Short match work, not just exact match */
-        { "alsa",   ITEM_TYPE_CARD },
-        { "atsc",   ITEM_TYPE_CARD },
-        { "bd",     ITEM_TYPE_DISC },
-        { "cable",  ITEM_TYPE_CARD },
-        { "cdda",   ITEM_TYPE_CDDA },
-        { "cqam",   ITEM_TYPE_CARD },
-        { "dc1394", ITEM_TYPE_CARD },
-        { "dccp",   ITEM_TYPE_NET },
-        { "deckli", ITEM_TYPE_CARD }, /* decklink */
-        { "dir",    ITEM_TYPE_DIRECTORY },
-        { "dshow",  ITEM_TYPE_CARD },
-        { "dv",     ITEM_TYPE_CARD },
-        { "dvb",    ITEM_TYPE_CARD },
-        { "dvd",    ITEM_TYPE_DISC },
-        { "dtv",    ITEM_TYPE_CARD },
-        { "eyetv",  ITEM_TYPE_CARD },
-        { "fd",     ITEM_TYPE_UNKNOWN },
-        { "ftp",    ITEM_TYPE_NET },
-        { "http",   ITEM_TYPE_NET },
-        { "icyx",   ITEM_TYPE_NET },
-        { "imem",   ITEM_TYPE_UNKNOWN },
-        { "itpc",   ITEM_TYPE_NET },
-        { "jack",   ITEM_TYPE_CARD },
-        { "linsys", ITEM_TYPE_CARD },
-        { "live",   ITEM_TYPE_NET }, /* livedotcom */
-        { "magnet", ITEM_TYPE_NET },
-        { "mms",    ITEM_TYPE_NET },
-        { "mtp",    ITEM_TYPE_DISC },
-        { "ofdm",   ITEM_TYPE_CARD },
-        { "oss",    ITEM_TYPE_CARD },
-        { "pnm",    ITEM_TYPE_NET },
-        { "qam",    ITEM_TYPE_CARD },
-        { "qpsk",   ITEM_TYPE_CARD },
-        { "qtcapt", ITEM_TYPE_CARD }, /* qtcapture */
-        { "raw139", ITEM_TYPE_CARD }, /* raw1394 */
-        { "rt",     ITEM_TYPE_NET }, /* rtp, rtsp, rtmp */
-        { "satell", ITEM_TYPE_CARD }, /* sattelite */
-        { "screen", ITEM_TYPE_CARD },
-        { "sdp",    ITEM_TYPE_NET },
-        { "sftp",   ITEM_TYPE_NET },
-        { "shm",    ITEM_TYPE_CARD },
-        { "smb",    ITEM_TYPE_NET },
-        { "svcd",   ITEM_TYPE_DISC },
-        { "tcp",    ITEM_TYPE_NET },
-        { "terres", ITEM_TYPE_CARD }, /* terrestrial */
-        { "torrent",ITEM_TYPE_NET },
-        { "udp",    ITEM_TYPE_NET },  /* udplite too */
-        { "unsv",   ITEM_TYPE_NET },
-        { "usdigi", ITEM_TYPE_CARD }, /* usdigital */
-        { "v4l",    ITEM_TYPE_CARD },
-        { "vcd",    ITEM_TYPE_DISC },
-        { "window", ITEM_TYPE_CARD },
+        { "alsa",    ITEM_TYPE_CARD },
+        { "atsc",    ITEM_TYPE_CARD },
+        { "bd",      ITEM_TYPE_DISC },
+        { "cable",   ITEM_TYPE_CARD },
+        { "cdda",    ITEM_TYPE_CDDA },
+        { "cqam",    ITEM_TYPE_CARD },
+        { "dc1394",  ITEM_TYPE_CARD },
+        { "dccp",    ITEM_TYPE_NET },
+        { "deckli",  ITEM_TYPE_CARD }, /* decklink */
+        { "dir",     ITEM_TYPE_DIRECTORY },
+        { "dshow",   ITEM_TYPE_CARD },
+        { "dv",      ITEM_TYPE_CARD },
+        { "dvb",     ITEM_TYPE_CARD },
+        { "dvd",     ITEM_TYPE_DISC },
+        { "dtv",     ITEM_TYPE_CARD },
+        { "eyetv",   ITEM_TYPE_CARD },
+        { "fd",      ITEM_TYPE_UNKNOWN },
+        { "ftp",     ITEM_TYPE_NET },
+        { "http",    ITEM_TYPE_NET },
+        { "icyx",    ITEM_TYPE_NET },
+        { "imem",    ITEM_TYPE_UNKNOWN },
+        { "itpc",    ITEM_TYPE_NET },
+        { "jack",    ITEM_TYPE_CARD },
+        { "linsys",  ITEM_TYPE_CARD },
+        { "live",    ITEM_TYPE_NET }, /* livedotcom */
+        { "magnet",  ITEM_TYPE_NET },
+        { "mms",     ITEM_TYPE_NET },
+        { "mtp",     ITEM_TYPE_DISC },
+        { "ofdm",    ITEM_TYPE_CARD },
+        { "oss",     ITEM_TYPE_CARD },
+        { "pnm",     ITEM_TYPE_NET },
+        { "qam",     ITEM_TYPE_CARD },
+        { "qpsk",    ITEM_TYPE_CARD },
+        { "qtcapt",  ITEM_TYPE_CARD }, /* qtcapture */
+        { "raw139",  ITEM_TYPE_CARD }, /* raw1394 */
+        { "rt",      ITEM_TYPE_NET }, /* rtp, rtsp, rtmp */
+        { "satell",  ITEM_TYPE_CARD }, /* sattelite */
+        { "screen",  ITEM_TYPE_CARD },
+        { "sdp",     ITEM_TYPE_NET },
+        { "sftp",    ITEM_TYPE_NET },
+        { "shm",     ITEM_TYPE_CARD },
+        { "smb",     ITEM_TYPE_NET },
+        { "svcd",    ITEM_TYPE_DISC },
+        { "tcp",     ITEM_TYPE_NET },
+        { "terres",  ITEM_TYPE_CARD }, /* terrestrial */
+        { "torrent", ITEM_TYPE_NET },
+        { "udp",     ITEM_TYPE_NET },  /* udplite too */
+        { "unsv",    ITEM_TYPE_NET },
+        { "usdigi",  ITEM_TYPE_CARD }, /* usdigital */
+        { "v4l",     ITEM_TYPE_CARD },
+        { "vcd",     ITEM_TYPE_DISC },
+        { "window",  ITEM_TYPE_CARD },
     };
     const struct item_type_entry *e;
 
-- 
2.3.1

