From 1ebdb15cdaec2bb442a18cda120a02155eb1f350 Mon Sep 17 00:00:00 2001
From: Jonathan Calmels <jbjcalmels@gmail.com>
Date: Mon, 16 Feb 2015 22:08:32 +0100
Subject: [PATCH 3/5] text: deduce magnet links as if the scheme was provided

---
 src/text/url.c | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/src/text/url.c b/src/text/url.c
index f2d7fb9..ea02d25 100644
--- a/src/text/url.c
+++ b/src/text/url.c
@@ -164,6 +164,17 @@ char *vlc_path2uri (const char *path, const char *scheme)
 
     char *buf;
 
+    if (scheme == NULL && !strncmp (path, "magnet:?", 8)) {
+        char hash_type[16];
+        char hash[64];
+
+        if (sscanf(path + 8, "xt=urn:%15[a-z]:%63[a-zA-Z0-9]", hash_type, hash) == 2) {
+            if (asprintf (&buf, "%s://%s", "magnet", path) == -1)
+                buf = NULL;
+            return buf;
+        }
+    }
+
 #ifdef __OS2__
     char p[strlen (path) + 1];
 
-- 
2.1.4

